react/AppLinker/__snapshots__/index.deprecated.spec.jsx.snap

```diff
@@ -2,23 +2,24 @@
 
 exports[`app icon should not crash if no href 1`] = `
 <div>
-  <a
-    onClick={[Function]}
-  >
-    Open 
-    Cozy Drive
-  </a>
+  <div>
+    <a>
+      Open 
+      Cozy Drive
+    </a>
+  </div>
 </div>
 `;
 
 exports[`app icon should render correctly 1`] = `
 <div>
-  <a
-    href="https://fake.link"
-    onClick={null}
-  >
-    Open 
-    Cozy Drive
-  </a>
+  <div>
+    <a
+      href="https://fake.link"
+    >
+      Open 
+      Cozy Drive
+    </a>
+  </div>
```

Comment:
is is attented to have a new `<div>` node? 

Also, are you sure that `onClick` is not a valid DOM Event for an `a`?

Reply:
> is is attented to have a new `<div>` node?

This is because we now snapshot the `container` from testing-library's `render()`.

`containers` add the `<div>` around the component.

> Also, are you sure that onClick is not a valid DOM Event for an a?

I'm not sure about what defines if a prop is kept or not.

For exemple the following test : 
```js
    const { container } = render(
      <div>
        <a href='http://someHREF' onClick='onClick' someOtherProp='somePROP'>
          Open
        </a>
      </div>
    )
    expect(container).toMatchSnapshot()
```

Produces the following snapshot:
```
exports[`SOME TEST`] = `
<div>
  <div>
    <a
      href="http://someHREF"
      someotherprop="somePROP"
    >
      Open
    </a>
  </div>
</div>
`;
```

The result is the same with `onclick=` (all lowercase like in w3c spec)

Reply:
<img width="432" alt="Capture d’écran 2022-08-08 à 16 30 00" src="https://user-images.githubusercontent.com/1107936/183442435-3c80e200-b444-4253-951d-0d905ca9551b.png">
<img width="328" alt="Capture d’écran 2022-08-08 à 16 30 41" src="https://user-images.githubusercontent.com/1107936/183442443-616c4a1a-088d-4976-9552-fe7864738ffb.png">
<img width="260" alt="Capture d’écran 2022-08-08 à 16 30 44" src="https://user-images.githubusercontent.com/1107936/183442444-c233d96e-2cd8-4899-b99c-aec21407742c.png">


So the DOM representation is not having the onclick attached. if it's specified later?